EAD-Session4/src/com/metacube/Parking/controller/Logout.java

```diff
@@ -0,0 +1,34 @@
+/**
+* This Servlet is a controller to logout user
+* @author Abhishek Bishnoi
+* @since Aug 29,2019
+*/
+package com.metacube.Parking.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+/**
+* invalidating session for logging out user
+*/
+public class Logout extends HttpServlet {
```

Comment:
We should have created a single LoginController, which should handle Login, LoginSuccess and Logout functionality in 3 different methods.